angular_bind_polymer.js

```diff
@@ -2,6 +2,7 @@ angular.module('eee-c.angularBindPolymer', []).
 directive('bindPolymer', function() {
   return {
     restrict: 'A',
+    scope: {}, /* Scope added here */
```

Comment:
Why not fire events from your polymer components and then use existing (`ng-click`, etc.) or self made directives for this? And if you want a two-way binding with setters and getters, you can use  `Object.defineProperty` to create an attribute with get/set: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty.